src/index.tsx

```diff
@@ -146,21 +147,24 @@ export const Netwatch: React.FC<IProps> = (props: IProps) => {
   return (
     <ThemeContext.Provider value={_theme}>
       <Modal animationType="slide" visible={visible} onRequestClose={handleBack}>
-        {showDetails ? (
-          <Details testId="detailScreen" onPressBack={setShowDetails} item={item} />
-        ) : (
-          <Main
-            maxRequests={props.maxRequests}
-            testId="mainScreen"
-            onPressClose={() => setVisible(false)}
-            onPressDetail={setShowDetails}
-            onPress={setItem}
-            reduxActions={reduxActions}
-            rnRequests={rnRequests}
-            nRequests={nRequests}
-            clearAll={clearAll}
-          />
-        )}
+        <View style={{ flex: 1 }}>
+          <View style={{ height: showDetails ? 0 : '100%' }}>
+            <Main
+              maxRequests={props.maxRequests}
+              testId="mainScreen"
+              onPressClose={() => setVisible(false)}
+              onPressDetail={setShowDetails}
+              onPress={setItem}
+              reduxActions={reduxActions}
+              rnRequests={rnRequests}
+              nRequests={nRequests}
+              clearAll={clearAll}
+            />
+          </View>
+          <View style={{ height: showDetails ? '100%' : 0 }}>
```

Comment:
We can keep both items with height set to 100%, and add an if condition on detail to display it or not ( with absolute position)